nbs/examples/Introduction.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<p>This is a naive, not and ETS. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>thanks</p>